Makefile

```diff
@@ -224,7 +224,11 @@ proto: check-tools-external
 	protoc-go-inject-tag -input=./helper/identity/types.pb.go
 	protoc-go-inject-tag -input=./helper/identity/mfa/types.pb.go
 
-fmt:
+importfmt:
```

Comment:
Do we need a new target distinct from `fmt`?  Is there a good use case for where we'd want to do only one but not the other?

Reply:
Who knows what the future might bring? This way we get the flexibility of both. Is there a downside to doing it this way?

Reply:
I agree, I think it makes sense to do both as part of `fmt`.

Reply:
A very minor downside: the Makefile already has a huge number of targets, so you're slightly increasing the complexity and decreasing its readability.  I don't feel strongly on this point.

Reply:
I think the readability-of-the-Makefile ship sailed a long time ago 😆. Since we're splitting hairs anyway though, I could argue that having it as a separate target means in the event that only the imports need to be fixed, and that code itself doesn't, having a separate target means the task will complete in half the time.

Reply:
Sure, but running either of `make fmt` or `make importfmt` is going to be slow regardless, so I don't expect people to use it in their normal workflows, they're going to modify their IDEs to prevent introducing the bad formatting in the first place.

Reply:
Oh for sure, I just meant that eventually I expect all repeat contributors to the codebase to adopt this approach.  And that if anyone complains about formatting via make being slow, we can point out that there's a better way.